src/sha256sum.zig

```diff
@@ -0,0 +1,30 @@
+const std = @import("std");
+const stdout = &std.io.getStdOut().outStream().stream;
+
+pub fn sha256(name: []const u8) !void {
+    // encoding function
```

Comment:
Maybe you should implement sha256 here?

Reply:
Still waiting for this feature! Super excited!